steps/PageSlugs.ts

```diff
@@ -116,7 +116,8 @@ export const repeatingStepSlugs = [
   PageSlugs.TenantPhoto,
   PageSlugs.NextOfKin,
   PageSlugs.Carer,
-  PageSlugs.OtherSupport
+  PageSlugs.OtherSupport,
+  PageSlugs.Verify
```

Comment:
I don't think `Verify` belongs here. What motivated adding it?

Reply:
i think these have snuck in somehow from another branch. Sorry.